.circleci/config.yml

```diff
@@ -551,282 +547,282 @@ workflows:
           pg_major: 14
-          image_tag: '14.0'
+          image_tag: '<< pipeline.parameters.pg14_version >>'
           make: check-isolation
           requires: [build-14]
       - test-citus:
           name: 'test-14_check-worker'
           pg_major: 14
-          image_tag: '14.0'
+          image_tag: '<< pipeline.parameters.pg14_version >>'
           make: check-worker
           requires: [build-14]
       - test-citus:
           name: 'test-14_check-operations'
           pg_major: 14
-          image_tag: '14.0'
+          image_tag: '<< pipeline.parameters.pg14_version >>'
           make: check-operations
           requires: [build-14]
       - test-citus:
           name: 'test-14_check-follower-cluster'
           pg_major: 14
-          image_tag: '14.0'
+          image_tag: '<< pipeline.parameters.pg14_version >>'
           make: check-follower-cluster
           requires: [build-14]
       - test-citus:
           name: 'test-14_check-columnar'
           pg_major: 14
-          image_tag: '14.0'
+          image_tag: '<< pipeline.parameters.pg14_version >>'
           make: check-columnar
           requires: [build-14]
       - test-citus:
           name: 'test-14_check-columnar-isolation'
           pg_major: 14
-          image_tag: '14.0'
+          image_tag: '<< pipeline.parameters.pg14_version >>'
           make: check-columnar-isolation
           requires: [build-14]
       - tap-test-citus:
           name: 'test_14_tap-recovery'
           pg_major: 14
-          image_tag: '14.0'
+          image_tag: '<< pipeline.parameters.pg14_version >>'
           suite: recovery
           requires: [build-14]
       - test-citus:
           name: 'test-14_check-failure'
           pg_major: 14
           image: citus/failtester
-          image_tag: '14.0'
+          image_tag: '<< pipeline.parameters.pg14_version >>'
           make: check-failure
           requires: [build-14]
 
       - test-arbitrary-configs:
           name: 'test-12_check-arbitrary-configs'
           pg_major: 12
-          image_tag: '12.8'
+          image_tag: '<< pipeline.parameters.pg12_version >>'
           requires: [build-12]
       - test-arbitrary-configs:
           name: 'test-13_check-arbitrary-configs'
           pg_major: 13
-          image_tag: '13.4'
+          image_tag: '<< pipeline.parameters.pg13_version >>'
           requires: [build-13]
       - test-arbitrary-configs:
           name: 'test-14_check-arbitrary-configs'
           pg_major: 14
-          image_tag: '14.0'
+          image_tag: '<< pipeline.parameters.pg14_version >>'
           requires: [build-14]
 
       - test-pg-upgrade:
           name: 'test-12-13_check-pg-upgrade'
           old_pg_major: 12
           new_pg_major: 13
-          image_tag: '12.8-13.4-14.0'
-          requires: [build-12,build-13]
+          image_tag: '<< pipeline.parameters.pg12_version >>-<< pipeline.parameters.pg13_version >>-<< pipeline.parameters.pg14_version >>'
```

Comment:
Minor: it would probably make sense to have a separate parameter for the postgres upgrade image

Reply:
I think upgrade tests should use the same versions with the rest of the tests.

If a developer wants to bump PG versions for the rest of the tests, but keep old versions for upgrades, they should explicitly update the values here. Otherwise the tests will fail, and IMO they should fail in this case.

Reply:
I guess Talha meant to have a parameter that concats those three params as a new variable to be used in upgrade tests.

Reply:
Yep, so that it becomes more readable